### PR TITLE
Minor changes to eb server to use Docker

### PR DESCRIPTION
**Pull request by bmccandless**
part of #1866

### CODE REVIEW

File: server/eb/app.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def get_csp_hashes(app, app_config):
 
 if __name__ == "__main__":
     try:
-        application.run(debug=debug, threaded=not debug, use_debugger=False)
+        application.run(host=app_config.server_config.app__host, debug=debug, threaded=not debug, use_debugger=False)
```
Comment (MDunitz) [inline server/eb/app.py:188]: is this dependent on changes in a different pr?

Comment (bmccandless) [reply server/eb/app.py:188]: No this is needed for docker.    I need to pass in "0.0.0.0" for the host so that it can server outside requests.

Comment (bmccandless) [reply server/eb/app.py:188]: There is a change in single-cell-infra that passes in host as 0.0.0.0

**Review by MDunitz — APPROVED** [review]
lgtm

**Pull request merged**